src/init1.cc

```diff
@@ -930,44 +928,6 @@ static cptr d_info_flags2[] =
 	"XXX1"
 };
 
-/*
- * Trap flags
- */
-static cptr t_info_flags[] =
```

Comment:
I think there may also be an array `k_info_flags2_trap` in this file which can now be removed.